CODEOWNERS

```diff
@@ -1,46 +1,97 @@
-scripts/              @ledgerhq/live-eng-core
-tools/                @ledgerhq/live-eng-core
-fastlane/             @ledgerhq/live-eng-core
+# How does it work? => https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+# Order is important; the last matching pattern takes the most
+# precedence. When someone opens a pull request that only
+# modifies JS files, only @js-owner and not the global
+# owner(s) will be requested for a review.
 
-# Live Blockchain Support Team
-apps/**/families/                                                    @ledgerhq/live-blockchain-support
-apps/ledger-live-desktop/src/live-common-set-supported-currencies.*  @ledgerhq/live-blockchain-support
-apps/ledger-live-mobile/src/live-common-setup.*                      @ledgerhq/live-blockchain-support
-libs/ledger-live-common/src/families                                 @ledgerhq/live-blockchain-support
-libs/ledgerjs/packages/cryptoassets                                  @ledgerhq/live-blockchain-support
-libs/ledgerjs/packages/types-cryptoassets                            @ledgerhq/live-blockchain-support
-libs/ledgerjs/packages/hw-app-*                                      @ledgerhq/live-blockchain-support
-libs/coin-*                                                          @ledgerhq/live-blockchain-support
-libs/domain-service                                                  @ledgerhq/live-blockchain-support
-ledgerjs/packages/evm-tools                                          @ledgerhq/live-blockchain-support
-docs                                                                 @ledgerhq/live-blockchain-support
+# Hub team
+## by default, hub manage apps and libs related to it & then we overrides many exceptions
+apps/ledger-live-desktop/                                @ledgerhq/live-hub
+apps/ledger-live-mobile/                                 @ledgerhq/live-hub
+libs/**/errors/                                          @ledgerhq/live-hub
+libs/**/logs/                                            @ledgerhq/live-hub
+libs/**/types-live/                                      @ledgerhq/live-hub
+libs/env/                                                @ledgerhq/live-hub
+libs/ledger-live-common/                                 @ledgerhq/live-hub
+libs/promise/                                            @ledgerhq/live-hub
+libs/ui/                                                 @ledgerhq/live-hub
 
+# Blockchain team
+.github/**/bot-*.yml                                     @ledgerhq/live-blockchain-support
+**/live-common-set*.ts                                   @ledgerhq/live-blockchain-support
+apps/**/families/                                        @ledgerhq/live-blockchain-support
+apps/cli/                                                @ledgerhq/live-blockchain-support
```

Comment:
I don't quite agree that we should maintain the CLI, being an app of Ledger Live.

I accept ownership by default, as we are using it and no one else is available to maintain, but there won't be much more maintenance than there is now.

Reply:
fair enough, if "cli" isn't considered as a product, the maintenance on it will be minimal. it's mostly a QA and dev tools.
there is nothing actively being developed there, so you will almost never be pinged on the PR related to it, if it happens to become a burden feel free to just remove that codeowners line, PR would then just not assign anyone anymore.

the reasoning of putting this explicit line was that the CLI mostly address blockchain topics by allowing to easily test implementations (the dev portal document it for coin integrators), blockchain team is currently the closest team (maybe the only team) that actively uses the cli afaik.
it has been collectively maintained from many devs, typically when a PR was about to break the CLI, due to incompatibility or change of live-common paths, devs also fixed it there (as the CI will ensure the cli compiles). So on top of being a command line version of Ledger Live, it also acts as being a safeguard.

Reply:
(also we can subdivive the cli into different ownerships, but if we do, we may want to do a CODEOWNERS in apps/cli – yes we can have multiple CODEOWNERS files!)